python/tidy/servo_tidy/tidy.py

```diff
@@ -549,6 +549,12 @@ def check_rust(file_name, lines):
             (r": &Vec<", "use &[T] instead of &Vec<T>", no_filter),
             # No benefit over using &str
             (r": &String", "use &str instead of &String", no_filter),
+            # There should be any use of banned types:
+            # Cell<JSVal>, Cell<JS<T>>, DOMRefCell<JS<T>>, DOMRefCell<HEAP<T>>
+            (r"(\s|:)?Cell<JSVal>", "Banned type Cell<JSVal> detected. Use MutJS<JSVal> instead", no_filter),
+            (r"(\s|:)?Cell<JS<.+>>", "Banned type Cell<JS<T>> detected. Use MutJS<JS<T>> instead", no_filter),
```

Comment:
As you may notice in the travis failure, this pattern also matches `UnsafeCell<Foo>`, which is not what we want. You can make sure tidy succeeds in the whole repo by running `./mach test-tidy --all`.